tools/cldr-apps/js/src/views/DashboardWidget.vue

```diff
@@ -31,11 +31,23 @@
         </button>
         <span v-for="catData of data.notifications" :key="catData.category">
           <template v-if="catData.total">
+            <input
+              type="checkbox"
+              :title="describeShow(catData.category)"
+              :id="'dash-cat-checkbox-' + '-' + catData.category"
```

Comment:
i don't know if there's a better idiom for this, but maybe there's a way to refactor the `:key` and `:id` calculations somehow?